oak-solr-osgi/pom.xml

```diff
@@ -48,6 +48,7 @@
                             org.apache.hadoop.*;resolution:=optional,
                             org.apache.regexp.*;resolution:=optional,
                             org.apache.log4j.*;resolution:=optional,
+                            org.apache.yetus.audience.*;resolution:=optional,
```

Comment:
According to the website, "Apache Yetus is a collection of libraries and tools that enable contribution and release processes for software projects."

That sounds weird. But OK...

Reply:
I know. As you can see here https://mvnrepository.com/artifact/org.apache.zookeeper/zookeeper/3.4.14 this was added as NON optional dependency. Without this declaration, osgi-it tests would fail.